r/tests/testthat/test-other-package-roundtrip.R

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+skip_on_cran()
+
+library(tibble)
+
+test_that("readr read csvs roundtrip", {
+  # indirection to avoid CHECK note and we don't otherwise need to Suggest this
+  pkg <- "readr"
+  skip_if(!requireNamespace(pkg, quietly = TRUE))
```

Comment:
This avoids `R CMD check` complaining on my local machine, which it does if I do `requireNamespace("readr", quietly = TRUE)` but is there a better way to do this?